src/test/scala/circtTests/ConventionSpec.scala

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package circtTests
+
+import chisel3._
+import circt.convention
+import circt.stage.ChiselStage
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+
+class ConventionSpec extends AnyFunSpec with Matchers {
+  private class PassThrough extends Module {
```

Comment:
Nit:
```suggestion
  private class PassThrough extends RawModule {
```

Using `RawModule` avoids creating the "implicit" clock and reset ports which are unused here.

Reply:
done, thanks